master/internal/rm/agentrm/provisioner/provisioner.go

```diff
@@ -143,4 +149,40 @@ func (p *Provisioner) provision(ctx *actor.Context) {
 			numToLaunch, p.provider.instanceType().Name())
 		p.provider.launch(ctx, numToLaunch)
 	}
+
+	telemetry.ReportProvisionerTick(ctx.Self().System(),
```

Comment:
In testing this - do we know the quantity of telemetry data produced relative to what we consume now? I wouldn't want this to suddenly flood the recipient system with a much higher frequency or volume of data than we're comfortable handling.

Reply:
realistically though, i think it is fine. idk if we have anyone actually submitting jobs at the pace of every 5s. if we limit it, i would limit it so a later refactor doesn't explode telemetry when it doesn't think provisioner tick == telemetry